plugins/Feedback/Feedback.php

```diff
@@ -121,15 +129,7 @@ public function getShouldPromptForFeedback()
         }
 
         // if is new user or old user field not exist
-        if ($nextReminderDate === false || $nextReminderDate <= 0) {
-
-            // if user is created more than 6 month ago, set reminder to today and show banner
-            $userCreatedDate = Piwik::getCurrentUserCreationData();
-            if (!empty($userCreatedDate) && Date::factory($userCreatedDate)->addMonth(6)->getTimestamp() < $now) {
-                $nextReminder = Date::now()->getStartOfDay()->subDay(1)->toString('Y-m-d');
-                $feedbackReminder->setUserOption($nextReminder);
-                return true;
-            }
+        if ($nextReminderDate === false) {
```

Comment:
This actually makes it worse, as everyone will now get notified 6 months after the update. I will merge this PR now nevertheless, so the other fixes are in `4.x-dev`. Will create a PR to improve those checks myself later